Dockerfile

```diff
@@ -51,15 +47,19 @@ RUN --mount=type=cache,target=/go/pkg/mod \
     --mount=type=cache,target=/root/.cache/go-build \
     CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} go build -ldflags="${LDFLAGS}" -gcflags="${GCFLAGS}" -o manager main.go
 
+FROM --platform=linux/${ARCH} bitnami/kubectl:1.31.1 AS kubectl
+
 FROM --platform=linux/${ARCH} registry.access.redhat.com/ubi8-micro:8.10
 
 ARG ARCH
 
 WORKDIR /
 COPY --from=builder /workspace/manager .
-COPY --from=builder /workspace/kubectl-${ARCH} /usr/local/bin/kubectl
 COPY --from=builder /workspace/crds /crds
 
+# Copy the kubectl binary
+COPY --from=kubectl /opt/bitnami/kubectl/bin/kubectl /usr/local/bin/
```

Comment:
We use UBI base image. bitnami/kubectl is debian-based. I don't think it's a good option to mix OS'ws

Reply:
Can you tell more about that?

Here we just copy the Kubectl binary. Kubectl is written in Golang, so it's self-contained. Did I miss anything?

Reply:
if we use only one binary it's ok